Fix(content): Eliminate Softlock in FW Pirates (#10531)
endless-sky/endless-sky: data/human/free worlds 1 start.txt

			`	Once you are outside, he pulls you aside and says, "Remember what we talked about, <first>." He walks back into the bar and leaves you alone.`
				accept

	on complete
		conversation
			`When you land on <origin>, you find the fleet that Tomek mentioned: a few militia heavy and medium warships, a couple of carriers, three squadrons of interceptors, and even four merchant ships of varying sizes. In total the fleet is nearly as large as one of the fleets that defended Sabik or Kornephoros, but as you survey the motley mix of grizzled militia veterans, young, naive interceptor pilots, and eager merchant captains, you can't help but wonder if a fleet of this size has a good chance against the strongest pirate world in Southern space.`
			`	The Navy failed to conquer Longjump. Why would a fleet of a similar size be capable of conquering Greenrock? Perhaps the Senate was right in suggesting a diplomatic solution. Or perhaps luck will be in your favor.`
			choice
				`	(Go ahead and attack Greenrock, disobeying the Senate's orders regardless of the consequences.)`
				`	(Give up on Tomek's scheme and follow the Senate's orders to make a diplomatic visit to Thule.)`
					goto thule

> 			`	You speak with some of the captains of the invasion fleet. Apparently Tomek has been busy visiting spaceports and giving rousing speeches about ending piracy and slavery once and for all in the South, hence the fleet being so large despite being unsanctioned by the Senate. That also explains the inclusion of a few merchant captains in the fleet, likely local captains who overheard the speeches and aren't interested in joining the Free Worlds to fight against the Navy, but are more than happy to make life easier for their freighters if this succeeds.`
> 			`	You plan to leave after nightfall, which should hopefully mean that Alondo doesn't realize that you've left him behind until it's too late. Now all that remains is for you to travel to <system>, defeat whatever opposition you find, and land on <planet>. Or die trying.`
> 			action
> 				event "battle for Greenrock"
> 			``

Reviewer: Put the action before the text, then we don't need an empty line.
We also need to set a condition here in order to use it immediately; the event will not set its condition until we take off.
```suggestion
			action
				event "battle for Greenrock"
				set "fw: will attack greenrock"
			`	You speak with some of the captains of the invasion fleet. Apparently Tomek has been busy visiting spaceports and giving rousing speeches about ending piracy and slavery once and for all in the South, hence the fleet being so large despite being unsanctioned by the Senate. That also explains the inclusion of a few merchant captains in the fleet, likely local captains who overheard the speeches and aren't interested in joining the Free Worlds to fight against the Navy, but are more than happy to make life easier for their freighters if this succeeds.`
			`	You plan to leave after nightfall, which should hopefully mean that Alondo doesn't realize that you've left him behind until it's too late. Now all that remains is for you to travel to <system>, defeat whatever opposition you find, and land on <planet>. Or die trying.`
```

> 				accept

Reviewer: Can't accept an on complete conversation.
This might not affect the behaviour, but it will almost certainly lead to confusion for readers.
```suggestion
				decline
```

	
	on offer
		conversation

> 			`When you land on <origin>, you find the fleet that Tomek mentioned: a few militia heavy and medium warships, a couple of carriers, three squadrons of interceptors, and even four merchant ships of varying sizes. In total the fleet is nearly as large as one of the fleets that defended Sabik or Kornephoros, but as you survey the motley mix of grizzled militia veterans, young, naive interceptor pilots, and eager merchant captains, you can't help but wonder if a fleet of this size has a good chance against the strongest pirate world in Southern space.`

Reviewer: ```suggestion
			`When you land on <origin>, you find the fleet that Tomek mentioned: a few militia heavy and medium warships, a carrier, three squadrons of interceptors, and even four merchant ships of varying sizes. In total the fleet is nearly as large as one of the fleets that defended Sabik or Kornephoros, but as you survey the motley mix of grizzled militia veterans, young, naive interceptor pilots, and eager merchant captains, you can't help but wonder if a fleet of this size has a good chance against the strongest pirate world in Southern space.`
```

	destination "Greenrock"
	clearance
	to offer

> 		has "event: battle for Greenrock"

Reviewer: Use the condition we set before, and leave the mission condition in for clarity.
```suggestion
		has "FW Pirates: Attack 1: done"
		has "fw: will attack greenrock"
```

		not "fw first chapter done"
		not "FW Refinery 1: offered"
	to fail

> 		has "event: battle for Greenrock"

Reviewer 2: the solution missed my brain by an inch
